services/static_sites.md

```diff
@@ -0,0 +1,25 @@
+# Static Sites
+
+## Jekyll
+
+Pros: widely-used static site builder, templating, customizable in Ruby, Github pages default  
+Cons: Ruby requires annual upgrades to keep current
+
+* [Cicognara Static](https://github.com/pulibrary/cicognara-static) (which builds a site using data
+  from [Cicognara Catalogo](https://github.com/pulibrary/cicognara-catalogo/)
+
+
+## Static HTML
+
+Pros: Absolute lowest maintenance effort, good way to sunset unsustainable apps that are inactive  
+Cons: Need existing HTML or some other tool/workflow to produce
+
+* [Capping Liberty exhibit](https://github.com/pulibrary/capping_liberty_exhibit_archive)
+* [Covid-19 &amp; Me Oral History Project](https://github.com/pulibrary/amp-up-training)
+* [Firestone Library rennovations site](https://github.com/pulibrary/firestone_renovations_archive)
+* [Milberg Collection of Irish Theater](https://github.com/pulibrary/milberg_exhibit_archive)
+* [Pathé Baby exhibit](https://github.com/pulibrary/pathebaby_exhibit_archive)
+* [Republic in the Wilderness](https://github.com/pulibrary/republic_exhibit_archive)
+* [Sin and the City: William Hogarth's London](https://github.com/pulibrary/hogarth_exhibit_archive)
+* [Thankful Nation exhibit](https://github.com/pulibrary/thankful_nation_exhibit_archive)
+* [Versailles Exhibit](https://github.com/pulibrary/versailles_exhibit_archive)
```

Comment:
Add https://github.com/pulibrary/james_lyon_exhibit_archive and https://github.com/pulibrary/njmaps_exhibit_archive. These are jekyll sites.